server/utils/routes.js

```diff
@@ -1,3 +1,13 @@
+import basePathConfig from '../configs/basePathConfig'
+const basePath = basePathConfig || ''
+
 export const routes = {
   charts: '/charts',
+  logoutForbiddenRoute: `${basePath}/logout?e=forbidden`,
+  unauthorizedRoute: `${basePath}/logout?e=unauthorized`,
+  logout: `${basePath}/logout`,
+  resetPw: `${basePath}/login?e=resetpw`,
+  resetPwNoUser: `${basePath}/resetpw?e=nouser`,
+  resetPwError: `${basePath}/resetpw?e=db`,
+  resetPwErrorUnmatched: `${basePath}/resetpw?e=unmatched`,
```

Comment:
these can all be the same route and we can pass a different error.

```suggestion
  loginWithError: (err) => `${basePath}/login?e=${err}`,
  resetWithError: (err) => `${basePath}/resetpw?e=${err}`,
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241/commits/be3741398c54927adbe373fa7b03098b9ba0424c